packages/pipeline-editor/src/PipelineService.tsx

```diff
@@ -27,6 +27,8 @@ export const RUNTIMES_NAMESPACE = 'runtimes';
 export const RUNTIME_IMAGES_NAMESPACE = 'runtime-images';
 export const COMPONENTS_NAMESPACE = 'components';
 
+const ELYRA_PREFIX_LEN = 6;
```

Comment:
I'm not sure I like this way of handling the prefix, @bourdakos1 this is different from how pipeline-editor handles it, should we do it like that side or just use this implementation?

Reply:
We can’t just strip off the first 6 characters, only a subset of the property IDs will be prefixed so we need to check that the property actually has the prefix

Reply:
My next update will change this to use a util function similar to how pipeline-editor handles it